### PR TITLE
Fix data validation in Excel interface, and exempt processing of input that is optional

### PR DESCRIPTION
**Pull request by nsmader**
These are fixed related to issues identified by CELFE staff.

### CODE REVIEW

File: run--03b--postestimation-disaggregate-estimates.Rmd [hunk 1]
```diff
@@ -350,7 +350,7 @@ add_agg_level <- function(ests, agg_level) {
         geo_crosswalk %>% 
         transmute(GEOID    = GEOID, 
                   County   = County, 
-                  geo_part = pct_area) %>% 
+                  geo_part = 1) %>%
```
Comment (kang-hyein) [inline run--03b--postestimation-disaggregate-estimates.Rmd:353]: Not sure why this has to be changed to 1 -- I know we need to fix the pct_area for county. Is this a temporary fix?

File: Interface for Specifying Custom ELPEP Eligibility Estimates.xlsm
Comment (kang-hyein) [inline Interface for Specifying Custom ELPEP Eligibility Estimates.xlsm:1]: Just checking which part of this Interface file was changed. I downloaded and compared to the old one and couldn't find any difference.

File: run--03b--postestimation-disaggregate-estimates.Rmd (hunk 1 above)
Comment (nsmader) [reply run--03b--postestimation-disaggregate-estimates.Rmd:353]: It's at least a temporary fix, yes. But: isn't it the case that tracts tend to line up perfectly with county boundaries?

File: Interface for Specifying Custom ELPEP Eligibility Estimates.xlsm
Comment (nsmader) [reply Interface for Specifying Custom ELPEP Eligibility Estimates.xlsm:1]: Hah! Good point -- these were invisible changes. Specifically, I fixed data validation where the wrong cell (one about max age) was requiring the value to be one of the values listed in the cell above (this was supposed to apply to determining which income eligibility threshold was the baseline one, from among the list above).

In a nice way -- it's the convenience of looking at diffs nice? Text readable files are so nice that way... thank goodness we're past the time of life that ipython and Jupyter notebooks were popular.

File: run--03b--postestimation-disaggregate-estimates.Rmd (hunk 1 above)
Comment (kang-hyein) [reply run--03b--postestimation-disaggregate-estimates.Rmd:353]: For just the county, yes. But when we incorporate the aux geo (CCA) I found an issue when separating the county into Chicago vs non-Chicago (there's an email thread with Bryan while you were on vacation). We should revisit that issue later.

**Comment by kang-hyein** [comment]
Review done and merging to main.

**Review by kang-hyein — APPROVED** [review]

**Pull request merged**